### PR TITLE
[Coming Soon] Update "Private" copy to "Coming Soon"

### PR DESCRIPTION
**Pull request by adamziel**
#### Changes proposed in this Pull Request

This PR updates site status badge to say "Coming soon" for appropriate sites. More context available here: paObgF-Ui-p2

Zrzut ekranu 2020-01-15 o 15 29 46

#### Testing instructions

1. Apply D37570-code
1. Set one of your simple sites to private
1. Also set it to "Coming Soon" mode by setting `wpcom_coming_soon` option to 1
1. Confirm the badge says "Coming Soon"
1. Confirm badges for other, private sites still say "Private"

### CODE REVIEW

File: client/blocks/site/index.jsx [hunk 1]
```diff
@@ -151,9 +151,14 @@ class Site extends React.Component {
 								: site.domain }
 						</div>
 						{ /* eslint-disable wpcalypso/jsx-gridicon-size */ }
-						{ this.props.site.is_private && (
-							<span className="site__badge site__badge-private">{ translate( 'Private' ) }</span>
-						) }
+						{ this.props.site.is_private &&
+							( this.props.site.is_coming_soon ? (
+								<span className="site__badge site__badge-private">
+									{ translate( 'Coming soon' ) }
```
Comment (a8ci18n) [inline client/blocks/site/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **62** times:
`translate( 'Coming Soon' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feature/coming-soon-support)

File: client/state/selectors/is-coming-soon-site.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import isPrivateSite from 'state/selectors/is-private-site';
+
+/**
+ * Returns true if the site is coming_soon
+ *
+ * @param {object} state Global state tree
+ * @param {object} siteId Site ID
+ * @returns {boolean} True if site is coming_soon
+ */
+export default function isComingSoonSite( state, siteId ) {
```
Comment (jblz) [inline client/state/selectors/is-coming-soon-site.js]: Minor: I'd change these names around in the function & file names (Is Site Coming Soon vs. Is Coming Soon Site).  It just reads better to me, personally. 

```suggestion
export default function isSiteComingSoon( state, siteId ) {
```

Comment (adamziel) [reply client/state/selectors/is-coming-soon-site.js]: I agree, I only went for `isComingSoonSite` to be consistent with `isPrivateSite`. I'd like both better is `isSiteCominigSoon` and `isSitePrivate` though!

Comment (michaeldcain) [reply client/state/selectors/is-coming-soon-site.js]: I started to make this change (and the `isSitePrivate` one), but it's a slippery slope. We follow both conventions in our code 🙄(i.e. `isJetpackSite` or `isSiteStore`). I do think that `is-site-` makes for a cleaner file structure.

I'll push the changes, but I don't mind if we just drop the commits if we decide against that approach.

Comment (michaeldcain) [reply client/state/selectors/is-coming-soon-site.js]: I opted not to finish the change for `isPrivateSite` to `isSitePrivate`. It's a pretty large undertaking and will junk up this PR. If we decide we want to pursue the change, we can do it in a follow-up.

Comment (adamziel) [reply client/state/selectors/is-coming-soon-site.js]: 👍  Thank you!

**Comment by adamziel** [comment]
I moved the selector part to another PR and may just merge it today since it's all new code: https://github.com/Automattic/wp-calypso/pull/38848

**Pull request merged**